src/webhook/test/webhook.factory.ts

```diff
@@ -0,0 +1,46 @@
+import { faker } from '@faker-js/faker';
+import { WebhookEventType } from '../dto';
+
+type CreateWebhookRequestOverrides = {
+  id?: number;
+  name?: string;
+  type?: WebhookEventType;
+  url?: string;
+  token?: string;
+};
+
+export const createWebhookRequest = (
+  overrides: CreateWebhookRequestOverrides = {},
+) => ({
+  name: overrides.name ?? faker.word.noun(),
+  type: overrides.type ?? WebhookEventType.RecipeCreated,
+  url: overrides.url ?? faker.internet.url(),
+  token: overrides.token ?? faker.string.alphanumeric(32),
+});
+
+export const createWebhookWithUserId = (
+  userId: number,
+  overrides: CreateWebhookRequestOverrides = {},
+) => ({
+  id: overrides.id ?? faker.number.int({ max: 65535 }),
+  name: overrides.name ?? faker.word.noun(),
+  type: overrides.type ?? WebhookEventType.RecipeCreated,
+  url: overrides.url ?? faker.internet.url(),
+  token: overrides.token ?? faker.string.alphanumeric(32),
+  userId,
+  initVector: faker.string.alphanumeric(16),
+  authTag: faker.string.alphanumeric(16),
+});
+
+export function createWebhookResponse() {
+  return {
+    id: faker.number.int({ max: 128 }),
```

Comment:
hmm, in `createWebhookWithUserId()` method you have `id: faker.number.int({ max: 65535 })` and in `createWebhookResponse()` `id: faker.number.int({ max: 128 })`. What are these `128` and `65535` values?

Reply:
There is an error, when faker generates too big Integers. 
```
"Unable to fit integer value '3313120212680704' into an INT4 (32-bit signed integer)."
```
so I hard coded those values to 'less bit' integers